compliance/google/unlabeled_resources/unlabeled_resources.pt

```diff
@@ -146,31 +147,40 @@ script "js_filter_resources", type: "javascript" do
     var obj = gr["all_res"];
     if (obj !== undefined && obj.items !== undefined) {
       for (var key in obj.items) {
-        var isResMissedTag = false;
+        var labelMissing = false;
         var all_include_label = '';
         var all_missed_label = '';
+        var all_missed_values = '';
         if (JSON.stringify(obj.items[key]["labels"]) == undefined) {
           var labels = "none";
         } else {
           var labels = obj.items[key]["labels"];
           var labels_str = JSON.stringify(labels);
-        }
+        }       
```

Comment:
Is this a tab added at the end of the line? If so I think we can remove it, otherwise ignore this comment.

Reply:
You are right. I removed it. Thanks!